.github/workflows/nmodl-doc.yml

```diff
@@ -0,0 +1,82 @@
+name: NMODL Documentation
+
+on:
+  push:
+    branches: 
+      - master
+      - live-debug*
```

Comment:
Forgot to update nmodl, would you mind removing this and adjusting the strategy from here: https://github.com/neuronsimulator/nrn/blob/master/.github/workflows/docs.yml#L65

same for `nmodl-ci.yml`